ncm-network/src/main/perl/nmstate.pm

```diff
@@ -518,6 +518,8 @@ sub nmstate_apply
             # apply config using nmstatectl
             my $ymlfile = $self->iface_filename($iface);
             if ($self->any_exists($ymlfile)){
+                $self->verbose("Apply config for inteface $iface");
```

Comment:
`Apply -> Applying` to be consist with e.g. line 515
and typo: inteface -> interface.